spec/requests/api/v1/restaurants_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::RestaurantsController, type: :request do
+
+  describe 'GET /v1/restaurants' do
+    it 'should return all the restaurant´s name and description' do
+      restaurant = create(:restaurant)
```

Comment:
This is okay for now but I usually prefer to create my objects using `let` or `let!` 
```
let(:restaurant) { create(:restaurant) }